web/themes/custom/cloudy/templates/field/file-link.html.twig

```diff
@@ -50,13 +50,23 @@
 {% set file_desc = icon_map[mime_type].desc ?? 'Download file'|t %}
 {% apply spaceless %}
 <div class="d-inline-flex">
-	<a href="{{ (document_link is empty) ? url : document_link }}">
+	{# If user is anonymous and the url is a webform submission upload,
```

Comment:
I made a small change to the twig template which wraps it in a parent <span> when there's no parent <a> tag, to avoid an alignment issue:

![image](https://github.com/eGovPDX/portlandor/assets/1520776/2c9f5361-2f11-4723-a824-e27f8ec9eff2)

I also shortened up the if statement a little bit (it seems to work correctly without the |render|render), and added a comment for future reference.